### PR TITLE
Intro to co2 data

### LINKED ISSUE #69: Introduction to the dataset and the problem
Starter doc https://ploomber-sql.readthedocs.io/en/latest/packaging-your-sql-project/introduction-to-the-dataset-and-problem.html

### PR DESCRIPTION
**Pull request by lfunderburk**
## Describe your changes

Introduction to the CO2 emissions dataset, and the upcoming topics

## Issue number

Closes #69 

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)

----
:books: Documentation preview :books:: https://ploomber-sql--74.org.readthedocs.build/en/74/

### CODE REVIEW

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md [hunk 1]
```diff
@@ -11,4 +11,42 @@ kernelspec:
...
+
+Additionally, they provide access through their [open data portal](https://open.canada.ca/data/en/dataset/98f1a129-f628-4ce4-b24d-6f16bf24dd64) as part of the [Open Government License Canada](https://open.canada.ca/en/open-government-licence-canada). 
+
+Whereas this tool allows consumers to obtain information via the website, it is desirable to develop an automated approach to extract information on the most recent models, and to be able to analyse the data to answer questions such as:
```
Comment (TegveerG) [inline docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md:22]: "Whereas this tool allows consumers...such as: " -> While this tool allows consumers to obtain information via the website, it is recommended to develop an automated approach to extract information on the most recent vehicle models and analyze the data to answer important questions:

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md [hunk 2]
```diff
@@ -11,4 +11,42 @@ kernelspec:
...
+
+Additionally, they provide access through their [open data portal](https://open.canada.ca/data/en/dataset/98f1a129-f628-4ce4-b24d-6f16bf24dd64) as part of the [Open Government License Canada](https://open.canada.ca/en/open-government-licence-canada). 
+
+Whereas this tool allows consumers to obtain information via the website, it is desirable to develop an automated approach to extract information on the most recent models, and to be able to analyse the data to answer questions such as: 
+
+1. What is the average CO2 emissions of vehicles by class, make and model?
+2. Are there vehicles better suited for city driving than others (focus on less CO2 emissions)?
+3. Are there vehicles better suited for highway driving than others (focus on less CO2 emissions)?
+4. Are there vehicles better suited for combined driving than others (focus on less CO2 emissions)?
```
Comment (TegveerG) [inline docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md:27]: I think points 2, 3, and 4 can be synthesized into a single point for eliminating some redundancy. I would recommend writing:
2. Which vehicles are better suited for different driving scenarios (city, highway, and combined) with a focus on lower CO2 emissions?

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md [hunk 3]
```diff
@@ -11,4 +11,42 @@ kernelspec:
...
+5. What are benefits of using hybrid vehicles over non-hybrid vehicles?
```
Comment (TegveerG) [inline docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md:28]: If we are synthesizing points 2, 3, and 4, we may want to add a few more (keeping points 1 and 5 as is). Here are some suggestions:

4. Can we identify trends in car manufacturing based on the number of vehicles produced per model year or by vehicle class (size) for different fuel types (gas, hybrid, and electric)?
5. How can fuel efficiency be analyzed for different fuel types (gas, hybrid, and electric)?

**Review by jpjon — COMMENTED** [review]
Agreed with Tegveer's suggestions, otherwise looks good!

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md (hunk 3 above)
Comment (lfunderburk) [reply docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md:28]: Pushed fix in last commit

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md (hunk 2 above)
Comment (lfunderburk) [reply docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md]: Pushed fix in last commit

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md (hunk 1 above)
Comment (lfunderburk) [reply docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md]: Pushed fix in last commit

File: docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md [hunk 4]
```diff
@@ -48,5 +48,5 @@ We will use the `requests` library to extract the metadata. We will then use `pa
 
 For the analysis, we will then use `SQL` to answer the questions above and `JupySQL`' functionality to perform exploratory data analysis, save interesting queries into the database, and later reuse these tables in the form of a dashboard.
```
[the remark this answers is not in the document]
Comment (TegveerG) [reply docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md:49]: @lfunderburk I think you wanted to write `JupySQL`'s functionality, the "s" is missing in the latest commit

[the remark this answers is not in the document]
Comment (lfunderburk) [reply docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md]: Fixed

**Review by jpjon — APPROVED** [review]

**Pull request merged**